web/packages/shared/components/UnifiedResources/CardsView/ResourceCard.tsx

```diff
@@ -340,11 +342,17 @@ const CardInnerContainer = styled(Flex)`
 `;
 
 const getBackgroundColor = props => {
+  if (props.requiresRequest && props.pinned) {
```

Comment:
nit: i'd probably extract this to a shared file b/c it looks the same one in `ResourceListItem`